wpilibNewCommands/src/main/native/include/frc2/command/CommandPtr.h

```diff
@@ -30,11 +33,11 @@ class CommandPtr final {
   explicit CommandPtr(std::unique_ptr<CommandBase>&& command)
       : m_ptr(std::move(command)) {}
 
-  template <class T, typename = std::enable_if_t<std::is_base_of_v<
-                         Command, std::remove_reference_t<T>>>>
+  template <std::derived_from<Command> T>
+  // NOLINTNEXTLINE (bugprone-forwarding-reference-overload)
   explicit CommandPtr(T&& command)
-      : CommandPtr(std::make_unique<std::remove_reference_t<T>>(
-            std::forward<T>(command))) {}
+      : CommandPtr(
+            std::make_unique<std::decay_t<T>>(std::forward<T>(command))) {}
```

Comment:
Is replacing `remove_reference_t` with `decay_t` relevant to this PR and functionally correct?

Reply:
Yes. It removes CV qualifiers as well as references.